vuepress/docs/next/docs/component-generator/component-gen-tech.md

```diff
@@ -22,10 +22,8 @@ When generating a microservice using the Entando Component Generator there are a
   * There are many powerful technologies in the marketplace but it is up to the developer to integrate any choices from the marketplace into their Entando application. Entando doesn't provide any direct support for marketplace tech choices.
 * Micro frontends
   * Users can optionally choose to generate micro frontends for entities generated via the blueprint.
-  * The Entando Component Generator only provides micro frontends in React from the generation flow but developers can create and bundle micro frontends in other technologies as well.
+  * The Entando Component Generator only provides micro frontends in React from the generation flow but developers can create and bundle micro frontends in other technologies as well. 
```

Comment:
would reorder and add comma; "The ECG generation flow only provides micro frontends in React, but developers..."

Reply:
"using other technologies" instead of "in other technologies"